azure.yaml

```diff
@@ -10,14 +10,27 @@ services:
     host: appservice
 hooks:
     postprovision:
-      shell: pwsh
-      run: ./scripts/prepdocs.ps1
-      interactive: true
-      continueOnError: false
+      windows:
+        shell: pwsh
+        run: ./scripts/prepdocs.ps1
+        interactive: true
+        continueOnError: false
+      posix:
+        shell: sh
+        run: ./scripts/postprovision.sh
```

Comment:
please name prepdocs.sh

Reply:
I'd like to avoid using the same name as the python script. This script only works in the context of a hook or in a shell where environment variables have been set from `azd env get-values`. It also does more/different things than the prepdocs.py script does (e.g. create the virtual environment)

Is there a specific reason that it should have the same name?

Reply:
Because it does the same thing, just a diff format.  This file won't always be run post-provision.  It is there to prep the docs.

Reply:
The primary purpose of `postprovision.sh` is to install the requirements for the `prepdocs.py` script/CLI. I expect users to run `prepdocs.py` directly to prep docs for subsequent runs. This is what allows them to specify which documents to index without copying them into the data folder etc.